src/de/tuebingen/uni/ub/hc/MARCProcessing/MARC4JProcessor.java

```diff
@@ -0,0 +1,153 @@
+package de.tuebingen.uni.ub.hc.MARCProcessing;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileWriter;
+import java.io.InputStream;
+
+import org.marc4j.MarcReader;
+import org.marc4j.MarcWriter;
+import org.marc4j.MarcXmlReader;
+import org.marc4j.MarcXmlWriter;
+import org.marc4j.marc.ControlField;
+import org.marc4j.marc.DataField;
+import org.marc4j.marc.Record;
+import org.marc4j.marc.Subfield;
+
+import de.tuebingen.uni.ub.hc.Corpus.IxTheoCorpus;
+import de.tuebingen.uni.ub.hc.Corpus.IxTheoRecord;
+import de.tuebingen.uni.ub.hc.enums.IxTheo_Annotation;
+
+public class MARC4JProcessor {
+	private static InputStream in;
+	private static IxTheoCorpus corpus;
+
+	public MARC4JProcessor(String pathname) throws FileNotFoundException {
+		in = new FileInputStream(pathname);
+		corpus = new IxTheoCorpus();
+		processMARCRecords();
+	}
+
+	public static void processMARCRecords() throws FileNotFoundException {
```

Comment:
You should just give this the arguments from the constructor and do away with the constructor altogether.